python/features/steps/general_purpose_steps.py

```diff
@@ -106,6 +126,31 @@ def check_model_fail(context, model_file):
         raise AssertionError(f"Check cli command succeeded with message: {output_message}")
     context.output_message = output_message
 
+@when('print-defs is called from terminal')
```

Comment:
I would see about genericizing this to get be any command that does not require a model file be provided, then it could be used for a plugin like `version` as well.

Reply:
So. should be able to get set up like the steps that get a model file but be the command passed in.